plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/areas/PreferenceArea.java

```diff
@@ -25,11 +25,11 @@
 import org.eclipse.ui.IWorkbench;
 
 /**
- * Abstract base class for preference areas. A preference area is somewhat like a
- * {@link org.eclipse.jface.preference.FieldEditor FieldEditor} except that it is expected to be
- * configured from an extension point.
- * <p>
- * Preference areas should notify its listeners of changes using property change events.
+ * Abstract base class for preference areas. A preference area is somewhat like a {@link
+ * org.eclipse.jface.preference.FieldEditor FieldEditor} except that it is expected to be configured
```

Comment:
This link may put a warning in the IDE since we don't import this class here. :-(

Reply:
The only actual change here was moving the `<p>` to the first line; the other changes are from the google-java-format.  We haven't had any warnings.

Reply:
I do get warnings on these in Eclipse. Maybe I'm configured a little differently.

Reply:
Odd.  And we now take preferences from the project settings, so we should see everything, though I suppose I'm running on Photon.  What's funny is that I remember putting in a FQN to avoid an unnecessary import warning, since it's only used in this javadoc.